app/src/sharedTest/java/org/oppia/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -79,6 +113,12 @@ private const val MORNING_TIMESTAMP = 1556094120000
 /** Tests for [NavigationDrawerTestActivity]. */
 @RunWith(AndroidJUnit4::class)
 @LooperMode(LooperMode.Mode.PAUSED)
+@Config(
+  application = NavigationDrawerTestActivityTest.TestApplication::class,
+  qualifiers = "port-xxhdpi"
+)
+// TODO(#973): Fix NavigationDrawerTestActivityTest
```

Comment:
Ditto here & elsewhere: maybe ignore the individual methods, instead?

Reply:
Yep I changed these cases to ignore individual methods (in these tests all the methods were failing so there are ignores for all of them)